src/Components/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,144 @@
+    if (typeof tabIndex === 'string') tabIndex = Number(tabIndex);
+    const tabPath = tabsPath[tabIndex];
+    if (tabPath !== undefined) {
+      navigate(tabPath);
+    }
+    setActiveTabKey(tabIndex);
+  };
+
+  return (
+    <React.Fragment>
+      <PageHeader>
+        <PageHeaderTitle className="title" title="Upgrades" />
+        <Popover
+          minWidth="35rem"
+          headerContent={'About upgrades'}
+          bodyContent={
+            <TextContent>
+              <Text>
+                This page helps perform an in-place upgrade of Red Hat Enterprise Linux. Separate tabs represents different phases user should go
+                through in order to succesfully upgrade system(s) to next major release.
+              </Text>
+              <Text>
+                <Button
+                  component="a"
+                  target="_blank"
+                  variant="link"
+                  icon={<ExternalLinkAltIcon />}
+                  iconPosition="right"
+                  isInline
+                  href={'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/upgrading_from_rhel_7_to_rhel_8/index'}
+                >
+                  Upgrading from RHEL 7 to RHEL 8
+                </Button>
+              </Text>
+              <Text>
+                <Button
+                  component="a"
+                  target="_blank"
+                  variant="link"
+                  icon={<ExternalLinkAltIcon />}
+                  iconPosition="right"
+                  isInline
+                  href={'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/upgrading_from_rhel_8_to_rhel_9/index'}
+                >
+                  Upgrading from RHEL 8 to RHEL 9
+                </Button>
+              </Text>
+            </TextContent>
+          }
+        >
+          <Button variant="plain" aria-label="About upgrades" className="pf-u-pl-sm header-button">
+            <HelpIcon />
+          </Button>
+        </Popover>
+      </PageHeader>
+      <Tabs className="pf-c-tabs pf-c-page-header pf-c-table" activeKey={activeTabKey} onSelect={handleTabClick}>
+        <Tab
+          eventKey={0}
+          title={<TabTitleText>Inventory</TabTitleText>}
+          actions={
+            // NOTE: Example of Help Popover - Update in OAMG-9571
+            <HelpPopover
+              header={'Inventory'}
+              body={
+                <div>
+                  <TextContent>
+                    <Text>Systems that are eligible for in-place upgrade using leapp utility.</Text>
+                    <Text>Systems have to be on supported upgrade paths</Text>
+                  </TextContent>
+                </div>
+              }
+            />
+          }
+        >
+          <section className="pf-l-page__main-section pf-c-page__main-section">TODO: Content Inventory</section>
```

Comment:
instead of "TODO: Content Inventory" we will include here the relevant inventory component from other file, correct? (in the next tasks)

Reply:
yeah, it's just examples of empty sections, I would have separate component for every tab that would be rendered in the section - I would keep section in the tabs, but it doesn't really matter, can also be inside the component itself

Reply:
sounds good